src/Adapter/Language/ContextLanguageDataProvider.php

```diff
@@ -24,7 +24,7 @@ public function __construct(LegacyContext $legacyContext)
      *
      * @return array
      */
-    public function getIncludingInactiveLocales()
+    public function getInstalledLocales()
```

Comment:
you cant' change public API :-1:

Reply:
I'm not sure I fully understand the problem here. I created here new adapter class called 'ContextLanguageDataPeovider' which has two public methods: 
'getActiveLocales' and 'getInstalledLocales' - I changed the name from 'getIncludingInactiveLocales' to 'getInstalledLocales' to keep it shorter.

The name is incorrect here or something else is wrong? Let me know, thanks ;)

Reply:
Is it a new class you created? forget my comment then :stuck_out_tongue: